### ISSUE TITLE
Extra bracket typo in error message

### ISSUE THREAD

**Issue by Betalos** [Dan503/mq-scss#5]
I'm getting this error while building scss project:
`
                @error 'Media type definitions must be specified at the start of plus statements: (#{$range)})';
                                                                                            ^
      Expected "}".
    ╷
307 │         @error 'Media type definitions must be specified at the start of plus statements: (#{$range)})';
    │                                                                                                    ^
    ╵
  node_modules\mq-scss\_mq.scss 307:94      root stylesheet
  projects\ft-core\style\_ft_core.scss 1:9  @import
  stdin 5:9                                 root stylesheet
      in C:\Users\thour\Documents\web-projects\ft-core-front\node_modules\mq-scss\_mq.scss (line 307, column 94)`

this is a syntax error in line 307:
`@error 'Media type definitions must be specified at the start of plus statements: (#{$range)})';`

**Update**

`#{$range)}` should be `#{$range}`
(There is an extra bracket)

**Comment by Dan503** [comment]
What's your input scss code?

**Comment by Betalos** [comment]
I recon it's just a syntax error that's strange it happened to me only on. Yet I am using you lib for more than a year.
The issue is in line 307 a ")" is added:
`
@error 'Media type definitions must be specified at the start of plus statements: (#{$range)})';
`
It should be: 
`
@error 'Media type definitions must be specified at the start of plus statements: (#{$range})';
`

**Comment by Dan503** [comment]
Yeah but what `@include mq(xxx)` statement did you write to make the error occur?

**Comment by Betalos** [comment]
Well I'm not sure which one, my app is an angular 7 one and the scss get maneged by the webpack loader.
The scss project is a large one and the mq is included multple times.
This is the full error stack:

```ERROR in ./src/styles.scss (./node_modules/@angular-devkit/build-angular/src/angular-cli-files/plugins/raw-css-loader.js!./node_modules/postcss-loader/src??embedded!./node_modules/sass-loader/lib/loader.js??ref--14-3!./src/styles.scss)
Module build failed (from ./node_modules/sass-loader/lib/loader.js):

                @error 'Media type definitions must be specified at the start of plus statements: (#{$range)})';
                                                                                            ^
      Expected "}".
    ╷
307 │         @error 'Media type definitions must be specified at the start of plus statements: (#{$range)})';
    │                                                                                                    ^
    ╵
  node_modules\mq-scss\_mq.scss 307:94      root stylesheet
  projects\ft-core\style\_ft_core.scss 1:9  @import
  stdin 5:9                                 root stylesheet
      in C:\Users\thour\Documents\web-projects\ft-core-front\node_modules\mq-scss\_mq.scss (line 307, column 94)
```

**Comment by Dan503** [comment]
I tried to make the error messages as helpful as I could since I couldn't guarantee that a line number would be produced where the error originates from. The idea being that you encounter the error when you enter the media query so you would know based on what the last mq you entered was.

If you really have no clue what mq is causing the error then you have no choice but to comment them all out one by one searching for `@include mq` until you find the culprit.

Until you know what the input scss is that is causing the error, I can't help you.

Based on the error message, it sounds like it is most likely an mq variable with a media type in it that is listed in a plus statement incorrectly in some way.

**Comment by Betalos** [comment]
Well I don't think it's an error with the mq include cuz in another project env the build is successfull with same import, the error accured when i made an update of the @angular/cli and i think it also updated webpack scss loader as well. maybe the loader now do a full code check cuz as i said it's just a syntax error, when i fixed the error the build succeed.

**Comment by Dan503** [comment]
Ahhh I see the bug in the error message now. It's a typo.

`#{$range)}` should be `#{$range}`

It has a bracket in the wrong place. Ok that is something that needs fixing in mq-scss so it outputs a propper clear error message.

**Comment by Dan503** [comment]
When it is outputting a propper error message then it will be easier to understand what is going wrong.

Unfortunately I'm unable to work on this for a couple of days.

**Comment by Dan503** [comment]
Oh I only just realised you pointed that out in an earlier comment https://github.com/Dan503/mq-scss/issues/5#issuecomment-477575120
😅

I was too focused on getting the input code from you.

**Comment by Betalos** [comment]
Hhh not a big deal , I should have poined explecitly the error instead of the line.
Well it's not of an urgent, like a sais it's still works (I don't know why).
Thank you for your quik replay bud 👍 .

**Comment by Dan503** [comment]
Fixed in version 2.1.5
